aspnetcore/mvc/views/razor.md

```diff
@@ -568,6 +568,31 @@ The code renders the following HTML:
 
 ::: moniker-end
 
+### @attribute
+
+The `@attribute` directive adds the given attribute onto the class of the generated view.
+
+```cshtml
+@attribute [Authorize]
+```
+
+> [!WARNING]
+> In 3.0.0-preview6 there is a known issue where `@attribute` directives do not work in `_Imports.razor` and `_ViewImports.cshtml`. This will be fixed in 3.0.0-preview7.
+
+### @namespace
+
+The `@namespace` directive sets the namespace of the class of the generated view.
+
+```cshtml
+@namespace Your.Namespace.Here
+```
+
+If a view imports something with a namespace directive then the original file's namespace is set relative to that namespace. 
+
+So if MyApp/Pages/_ViewImports.cshtml contains `@namespace Hello.World` then the namespace of views that import it will be set as follows:
+- MyApp/Pages/Index.cshtml => Hello.World
+- MyApp/Pages/MorePages/Bar.cshtml => Hello.World.MorePages
+
```

Comment:
There is missing info from @ajaybhargavb:
> And if for example, MyApp/Pages/Index.cshtml also contains a @namespace set, then that preferred. Basically the @namespace directive closest to the file wins.

and in my opinion it is important. Otherwise someone can think that imported namespace and namespace defined locally are concatenated.